pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -59,7 +60,16 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 	// create kubeadm init config
 	fns := []func() error{}
 
+	// TODO(bentheelder): settle on a better abstraction for this
+	// While we don't want much code aware of the provider name,
+	// we do want to surface it in the node providerID field
+	providerName := "docker"
```

Comment:
really hesitant to encourage further reliance on this, but I think it makes sense to surface this data in the providerID field and I want to ship that.
We can replace this with something else later (maybe a public method that isn't officially part of the interface or something, need to think on that)

Reply:
Perhaps `String()`

Reply:
Yeah, Stringer probably seems good enough.

Reply:
https://github.com/kubernetes-sigs/kind/pull/1809 done